doc/developer-guides/hld/hld-overview.rst

```diff
@@ -463,11 +467,12 @@ OVMF Bootloader
 Open Virtual Machine Firmware (OVMF) is the virtual bootloader that supports
 the EFI boot of the User VM on the ACRN hypervisor platform.
 
-The OVMF is copied to the User VM memory by the VM manager in the Service VM while creating
-the User VM virtual BSP of the User VM. The Service VM passes the start of OVMF and related
-information to HV. HV sets guest RIP of the User VM virtual BSP as the start of OVMF
-and related guest registers, and launches the User VM virtual BSP. The OVMF starts
-running in the virtual real mode within the User VM. Conceptually, OVMF is part of the User VM runtime.
+The OVMF is copied to the User VM memory by the VM Manager in the Service VM
+while creating the User VM virtual BSP of the User VM. The Service VM passes the
```

Comment:
"while creating" --> Because of the passive voice ("is copied by"), the grammatical subject of the sentence is _The OVMF_, so technically "while creating" refers to the OVMF (which I'm betting isn't correct). So a touch of rewriting is required here.